CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ __BACKWARDS INCOMPATIBILITIES:__
 
 IMPROVEMENTS:
  * discovery: Allow `check_restart` to be specified in the `service` stanza.
-   [GH-3718]
+   [[GH-3718](https://github.com/hashicorp/nomad/issues/3718)]
```

Comment:
Add description and subsystem this PR is for.

Reply:
I am going to back this out. This wasn't my change, make dev linkifies the changelog and it must have been missing a link.